packages/lisk-p2p/src/peer.ts

```diff
@@ -214,6 +214,19 @@ export class Peer extends EventEmitter {
 		}
 	}
 
+	public dropInboundConnection(code: number = 1000, reason?: string): void {
+		if (this._inboundSocket) {
+			this._inboundSocket.destroy(code, reason);
+			this._stopListeningForRPCs(this._inboundSocket);
```

Comment:
`_stopListeningForMessages`? also, probably we should stop listening before destroying

Reply:
That's true, I will also modify [disconnect()](https://github.com/LiskHQ/lisk-elements/blob/1a786675868225175bd77db05f2a91ebfb099cd3/packages/lisk-p2p/src/peer.ts#L207) function as well, as these functions are derived from it